src/wwwroot/app/services/VacancyService.js

```diff
@@ -104,13 +111,25 @@ export default class VacancyService {
          vacancy.responsibleId = parseInt(vacancy.responsibleId);
 
          if (vacancy.id) {
-            const additionalUrl = VACANCY_URL + vacancy.id;
-            return _HttpService.put(additionalUrl, vacancy);
+            return _HttpService.put(`${VACANCY_URL}${vacancy.id}`, vacancy);
          } else {
             return _HttpService.post(VACANCY_URL, vacancy);
          }
       }).then(this.convertFromServerFormat);
    }
+
+   _fillChildVacancies(vacancy) {
+      vacancy.childVacancies = [];
+      if (vacancy.childVacanciesIds.length) {
+         each(vacancy.childVacanciesIds, (childVacancyId) => {
+            return _VacancyService.getVacancy(childVacancyId).then((response) => {
```

Comment:
practically the same as in previous warn - use your return to decrease the curly braces nesting level